### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by clementAC**
- **remove tailwind warnings on colors**
- **minor deps upgrade**
- **fix warnings on blog authors**
- **Merge readme into one file**

A little bit of love for our doc stack. 
Fix all warnings during compile time, upgrade minor versions of our dependencies, merged the two readme files into one at the root of the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,29 @@
 
 This repository holds the documentation for the [crowdsec project](https://github.com/crowdsecurity/crowdsec).
 
-Online version of this documentation is available here: https://doc.crowdsec.net/
\ No newline at end of file
+Online version of this documentation is available here: https://doc.crowdsec.net/
+
+The documentation is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
```
Comment (LaurenceJJones) [inline README.md:7]: its v3 now

**Comment by LaurenceJJones** [comment]
Guess we merge this before remove top level directory?

**Comment by LaurenceJJones** [comment]
Some conflicts because of the add of additonal package, once conflicts have been solved we can merge this.

**Comment by LaurenceJJones** [comment]
I guess https://github.com/crowdsecurity/crowdsec-docs/pull/654 superceeds this?

**Comment by clementAC** [comment]
> I guess #654 superceeds this

No it's a PR train we can first merge this one and then the seconds one

**Pull request merged**